### PR TITLE
`columnNames` should allow a singular variant

### PR DESCRIPTION
**Pull request by SebastienFCT**
This was requested for the next milestone in https://github.com/cfwheels/cfwheels/issues/922

### CODE REVIEW

**Comment by SebastienFCT** [comment]
Thank you for the feedback and the example @chapmandu , I'll look into it and try to apply the changes ASAP.

**Comment by SebastienFCT** [comment]
@chapmandu I'm currently working on the UT and I noticed the following:

Calling a migrator method that is not available for a given database will create an error. For example:

```
t = migration.createTable(name=tableName, force=true);
t.bigInteger(columnName=columnName);
t.create();
```

works fine on `MySQL` but will fail for `PostgreSQL`, this is essentially because `Abstract.typeToSQL()` will return an empty string if the the variables doesn't contain the key "biginteger", which then will result in a SQL syntax error. 

Is there any way I can prevent this from happening in the UT? Skipping the case if I'm running on a specific DB?

Here's an example of the the test for adding multiple big integers:

```
function test_add_multiple_big_integers() {

	// Should I just break the test if `migration.adapter.adapterName()` == "PostgreSQL"` as postgres doesn't support big integer?

	tableName = "dbm_addbigInteger_tests";
	columnNames = "bigIntegerA,bigIntegerB";
	t = migration.createTable(name=tableName, force=true);
	t.bigInteger(columnNames=columnNames);
	t.create();

	...
}
```

**Comment by neokoenig** [comment]
We've generally avoided trying to test for too many db specific things, but there are msSQL exceptions already - see https://github.com/cfwheels/cfwheels/blob/master/wheels/tests/migrator/migration/createtable.cfc as an example: so I guess you could use `if !(migration.adapter.adapterName() eq "blah")`

**Comment by SebastienFCT** [comment]
Hey @neokoenig , thanks for the answer! I went with something similar. I'll update the pull request for review soon. 

There might be a little too much tests then (I added new file for testing specific functions integer(), boolean(), etc... and if that's the case I'll gladly remove them -still learnt a lot through them!-)

Cheers

**Comment by neokoenig** [comment]
Keep em in - the db migrate system was always a little under tested!

**Comment by chapmandu** [comment]
Great work my friend!

**Comment by chapmandu** [comment]
Be sure to immortalise yourself in the CHANGELOG.md in the miscellaneous section underneath the "validateTestPackageMetaData" entry.. Just use the same format as the others.

**Comment by SebastienFCT** [comment]
Done, thanks!

File: wheels/tests/migrator/addbinary.cfc [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+	private string function getBinaryType() {
+		switch (migration.adapter.adapterName()) {
+			case "H2":
+				return "BLOB";
+			case "MicrosoftSQLServer":
+				return "IMAGE";
+			case "MySQL":
+				return "BLOB";
+			case "PostgreeSQL":
```
[the remark this answers is not in the document]
Comment (SebastienFCT) [reply wheels/tests/migrator/addbinary.cfc]: Yes it was! I reviewed the other tests as well

**Pull request merged**